src/pyqasm/expressions.py

```diff
@@ -34,7 +34,8 @@
 
 from pyqasm.analyzer import Qasm3Analyzer
 from pyqasm.exceptions import ValidationError, raise_qasm3_error
-from pyqasm.maps import CONSTANTS_MAP, qasm3_expression_op_map
+from pyqasm.maps.gates import qasm3_expression_op_map
```

Comment:
`qasm3_expression_op_map` should be present in `pyqasm.maps.expressions.py`

Reply:
Done